Samples/Util/ZitiCommand.cs

```diff
@@ -56,7 +56,8 @@ public struct Options {
                 isSupported(supportedCmds, 7, "Generate MFA recovery codes for the identity");
                 isSupported(supportedCmds, 8, "Enable identity");
                 isSupported(supportedCmds, 9, "Disable identity");
-                isSupported(supportedCmds, 10, "Check identity enable status");
+                isSupported(supportedCmds, 10, "Check identity enabled status");
+                isSupported(supportedCmds, 11, "Invoke Endpoint satus change");
```

Comment:
typo: satus -> status

Reply:
Thanks for pointing out the typo. It has been corrected now.